src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -478,5 +479,42 @@ internal static void ExecuteViewDetailsSideBar(Step stepInfo, StepUIAutomation u
                 dynamoView.CloseExtensionTab(closeButton, null);
             }
         }
+
+        /// <summary>
+        /// This method will calculate the Popup location based in a item from the Library
+        /// </summary>
+        internal static void CalculateLibraryItemLocation(Step stepInfo, StepUIAutomation uiAutomationData, bool enableFunction, GuideFlow currentFlow)
+        {
+            CurrentExecutingStep = stepInfo;
+            if (uiAutomationData == null) return;
+            string jsFunctionName = uiAutomationData.JSFunctionName;
+            object[] jsParameters = new object[] { uiAutomationData.JSParameters[0] };
```

Comment:
use var instead of object[].

Reply:
@RobertGlobant20 Were you able to address this?

Reply:
@QilongTang fixed in the next commit: